corehq/apps/users/middleware.py

```diff
@@ -77,14 +77,23 @@ def process_view(self, request, view_func, view_args, view_kwargs):
         ):
             return None
 
-        if not toggles.TWO_FACTOR_SUPERUSER_ROLLOUT.enabled(request.user.username):
-            return None
-        elif request.user.is_staff or request.user.is_superuser and not request.user.is_verified():
-            if request.path.startswith('/account/') or request.couch_user.two_factor_disabled:
-                return None
+        return self.enforce_two_factor(toggles, request)
```

Comment:
you shouldn't have to pass `toggles` since this is a module import. Also not sure what the benefit of making this a separate method is?

Reply:
I separated it out so I could test just the function, is there a better way to do this?

Reply:
Added back to function, will just test process_view